Add Feedback/Rate Us section done ! #290

Subhajit-2023-44 opened the pull request:
Hello, PR
Add Feedback/Rate Us section with star rating widget - #290 
Done properly -


https://github.com/user-attachments/assets/24624dff-f436-4b54-8222-3cc82d6dca9b


![Screenshot 2024-10-31 030545](https://github.com/user-attachments/assets/c7be542f-7712-4488-89a6-3de0a7184446)
![Screenshot 2024-10-31 030554](https://github.com/user-attachments/assets/2f5bf082-d45a-4b1b-b0ef-72397617dae6)
![Screenshot 2024-10-31 030611](https://github.com/user-attachments/assets/ccbd9965-b609-4c3b-b1b6-d7b7d4cb4fe7)



## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [x] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

Subhajit-2023-44 commented:
@Harshdev098 sir done, please merge my PR with level 3, I also add a star rating widget with feedback form..


Thank You

Subhajit-2023-44 commented:
@Harshdev098 sir

Subhajit-2023-44 commented:
@Harshdev098 sir

Harshdev098 reviewed (changes requested):
I think you can make it more responsive

![image](https://github.com/user-attachments/assets/5c782e5d-bf1e-4cf4-98fe-4b2ce6e1915d)

Subhajit-2023-44 commented:
> I think you can make it more responsive


What ? I didn't understand ... 

@Harshdev098 sir i added everything what i can do, Feedback form, toast message & also star rating widget...

![Screenshot 2024-10-31 030554](https://github.com/user-attachments/assets/2f5bf082-d45a-4b1b-b0ef-72397617dae6)

If you check other projects, they give level 2/3 for feedback form and level 2 for star rating and another level 2 for toast message but I added everything in this PR because tomorrow was the last day, so...

@Harshdev098 sir please merge my PR 🙏🏻

Harshdev098 commented on public/css/style.css:
```diff
@@ -2241,3 +2242,174 @@ body {
+  cursor: pointer;
+  margin-left: 183px;
+  margin-top: -33px; /* Add space above the button */
+  display: block; /* Ensure it takes the full width */
```
Make the display, `flex` and add some padding in x-axis of the `i` tag

Subhajit-2023-44 replied:
> Make the display, `flex` and add some padding in x-axis of the `i` tag

😢okay wait i will give a new PR again

Subhajit-2023-44 commented:
@Harshdev098 sir done #375
